### ISSUE TITLE
Nested update with `updateMany` does not update the updatedAt timestamps for related records properly

### ISSUE THREAD

**Issue by mick703** [prisma/prisma#8265]
### Bug description

Hi, we found an issue with the updated_at timestamp not getting updated when we use `updateMany` in a nested update. When we update order with its children order lines by using the nested update, the timestamps `updated_at` on the order lines do not get updated automatically with `@updatedAt` specified in the schema. I do not see an issue reported in here. Did I miss anything or this is a known issue? 

### How to reproduce

- Create an order with an order line 
- Use `prisma.order.update` to update the order with nested update `updateMany` to update its order line records, eg, update the `sku` to have a new value.
- Retrive the order line record by prisma findFirst. 

Here is the test cases, 

```typescript
  describe('prisma client', () => {
    it('prisma should update updated_at timestamp properly with updateMany', async () => {
      const order = Order.create(allOrderProps()).getValue();
      await repo.save(order);
      const prismaOrderCreated = await prisma.order.findFirst({
        where: { id: order.id.toValue() },
        include: { order_lines: true }
      });
      expect(prismaOrderCreated.order_lines[0].updated_at).toBeDefined();

      //Use Prisma query to update the order line
      const orderLineId = prismaOrderCreated.order_lines[0].id;
      await prisma.orderLine.updateMany({
        where: { id: orderLineId },
        data: { external_id: 'updated external id' }
      });

      const prismaOrderLineUpdated = await prisma.orderLine.findFirst({ where: { id: orderLineId } });

      expect(prismaOrderLineUpdated.updated_at.getTime()).toBeGreaterThan(
        prismaOrderCreated.order_lines[0].updated_at.getTime()
      );
    });

    it('prisma updateMany should update updated_at timestamp for the related records properly', async () => {
      const order = Order.create(allOrderProps()).getValue();
      await repo.save(order);
      const prismaOrderCreated = await prisma.order.findFirst({
        where: { id: order.id.toValue() },
        include: { order_lines: true }
      });
      expect(prismaOrderCreated.order_lines[0].updated_at).toBeDefined();
      const orderLineId = order.orderLines.getItems()[0].id.toValue();

      await prisma.order.update({
        data: {
          email: 'updated@example.com',
          order_lines: {
            updateMany: {
              data: { external_id: 'updated_id' },
              where: { id: orderLineId }
            }
          }
        },
        where: { id: order.id.toValue() }
      });

      const prismaOrderLineUpdated = await prisma.orderLine.findFirst({ where: { id: orderLineId } });
      expect(prismaOrderLineUpdated.updated_at.getTime()).toBeGreaterThan(
        prismaOrderCreated.order_lines[0].updated_at.getTime()
      );
    });
  });

```

The first test passed while the second one failed as the updated_at field was not updated. 

### Expected behavior

The order line record should have an updated value for `updated_at` field when it was updated in the nested update. But the `updated_at` field in the order line record does not get updated accordingly even when the other fiedls are updated properly for the order line. 

### Prisma information

This is the schema,

```javascript
model Order {
  	id                                   String                  @id
  	external_id                          String? 
	... 
  	order_lines                          OrderLine[]
	created_at                           DateTime                @default(now())
  	updated_at                           DateTime                @default(now()) @updatedAt
}

model OrderLine {
	id 				  String 		    @id
	sku				  String
	external_id	      String
	...
    order_id          String
    created_at        DateTime          @default(now())
    updated_at        DateTime          @default(now()) @updatedAt
    order             Order             @relation(fields: [order_id], references: [id])
}
```

### Environment & setup

- OS: Mac OS, also on the AWS Lambda runtime
- Database: Postgres:11 and RDS Aurora 11.9
- Node.js version: 14.15.4 

### Prisma Version

```
prisma               : 2.24.1
@prisma/client       : 2.24.1
Current platform     : darwin
Query Engine         : query-engine 18095475d5ee64536e2f93995e48ad800737a9e4 (at node_modules/@prisma/engines/query-engine-darwin)
Migration Engine     : migration-engine-cli 18095475d5ee64536e2f93995e48ad800737a9e4 (at node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine : introspection-core 18095475d5ee64536e2f93995e48ad800737a9e4 (at node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary        : prisma-fmt 18095475d5ee64536e2f93995e48ad800737a9e4 (at node_modules/@prisma/engines/prisma-fmt-darwin)
Default Engines Hash : 18095475d5ee64536e2f93995e48ad800737a9e4
Studio               : 0.397.0
```

**Comment by mick703** [comment]
Thanks guys!

**Comment by dpetrick** [comment]
This will be fixed in 2.29.
